packages/flutter_locations/lib/src/ui/widgets/map.dart

```diff
@@ -119,7 +149,14 @@ class LocationsMap extends HookWidget {
                   initialZoom: initialZoom,
                   onMapReady: initializeMaps,
                 ),
-                children: layers,
+                children: [
+                  if (config.isFallbackMapType)
```

Comment:
the config should only be limited to the platform maps, and not be used outside. If it has global impact on the user-story it should be part of the options